src/main/java/seedu/address/model/person/Favorite.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Favorite status in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidFavoriteStatus(String)}
+ */
+public class Favorite {
+
+    /**
+     * Default favorite statuses
+     */
+    public enum FavoriteStatus {
+        yes,
+        no;
+    }
+
+    public static final String MESSAGE_FAVORITE_CONSTRAINTS =
+            "Favorite status can only be either 'yes' or 'no'";
+
+    public static final String FAVORITE_VALIDATION_REGEX = "yes|no";
+
+    public final String value;
```

Comment:
Can is store it as a boolean instead of a String, since it uses less space and leaves us less room for errors (e.g. "YES" vs "yes"?

Reply:
Agreed, implemented! 👍